p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4681,7 +4681,7 @@ It is illegal to instantiate a ```parser``` within a ```control```
 block.
 For a description of the ```optConstructorParameters```, which
 can be used to build parameterized control blocks, see Section
-[#sec-parametrization].
+[#sec-parameterization].
```

Comment:
Weird. I did a search-and-replace for this typo. Someone must have manually backed it out.